internal/cli/root/atlas/builder.go

```diff
@@ -75,8 +78,34 @@ type Notifier struct {
 	writer         io.Writer
 }
 
+func handleSignal(cmd *cobra.Command) {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+
+	go func() {
+		sig := <-c
+		telemetry.TrackCommand(telemetry.TrackOptions{
+			Cmd: cmd,
+			Err: errors.New(sig.String()),
+		})
+		os.Exit(1)
+	}()
```

Comment:
is there a chance of a race condition with https://github.com/mongodb/mongodb-atlas-cli/blob/a83ba08bcada62fbfbcd6cff4e613f372343802e/internal/cli/atlas/quickstart/quick_start.go#L308-L315 specially since both exit

Reply:
@gssbzn unfortunately there is no guarantee of an order (I'm reading some interesting article about it https://go.dev/ref/mem)

Safe to assume some logic must be changed here, I'll work towards a solution of having only one listener at a time